### ISSUE TITLE
[Enhancement]: Consider adding Ubuntu 23.04 packages

### ISSUE THREAD

**Issue by AlexanderPopovSLIC** [timescale/timescaledb#5834]
### What type of enhancement is this?

User experience

### What subsystems and features will be improved?

Packaging

### What does the enhancement do?

Ubuntu 23.04 was released on the April 20, 2023, can you please add builds for it? Postgres 15 now is available for Ubuntu 23.04 but TimescaleDB extension cannot be installed there.

**Comment by AlexanderPopovSLIC** [comment]
At the current moment, I have a temporary solution for me to be able to use TimescaleDB on Ubuntu 23.04 - is just to build it from sources and install. At least it works for me. Sorry for my Linux syntax, I'm not so friendly with UNIX systems :) but this script works for me. 
But it is better to wait for official packages. Would be great if it happen as soon as possible. Thanks.

```
# Install necessary libs to make it happen
sudo apt-get install git cmake postgresql-server-dev-15 libkrb5-dev golang -y
# cd to home directly
cd ~
# clone repository into home directory
sudo git clone -q https://github.com/timescale/timescaledb
# cd to timescaledb folder
cd timescaledb
# check the latest release number here - https://github.com/timescale/timescaledb/releases
# at the current moment it is 2.11.1 - so, lets checkout it
sudo git checkout 2.11.1
# run bootstrap script
sudo ./bootstrap
# cd to build folder
cd ./build
# run make command
sudo make
# run install command
sudo make install
# cd to home directly and remove the source code
cd ~
sudo rm -dR timescaledb/


## Tuning postgres config (can be skipped if tunning doesn't need)
# cd to home directly
cd ~
# clone timescaledb-tune repository
sudo git clone -q https://github.com/timescale/timescaledb-tune.git
# cd to the tune cmd folder
cd timescaledb-tune/cmd/timescaledb-tune/
# run the next command
sudo go run main.go --quiet --yes
# cd to home directly and remove the source code
cd ~
sudo rm -dR timescaledb-tune/



# remove libs
# !!! Warning !!! Check the list below because maybe some of libs are still in use and removing them is not necessary
sudo apt-get remove git -y && sudo apt-get purge git -y
sudo apt-get remove cmake -y && sudo apt-get purge cmake -y
sudo apt-get remove postgresql-server-dev-15 -y && sudo apt-get purge postgresql-server-dev-15 -y
sudo apt-get remove libkrb5-dev -y && sudo apt-get purge libkrb5-dev -y
sudo apt-get remove golang -y && sudo apt-get purge golang -y



# clean apt
sudo apt-get clean
sudo apt-get autoremove --purge
sudo apt-get autoclean
sudo apt -f install
sudo dpkg --configure -a

sudo apt update
```

**Comment by svenklemm** [comment]
I think this can be closed now since we package for 24.04. I think the policy is to package for latest ubuntu and ubuntu lts versions
